tools/cli/commands/creategpu.py

```diff
@@ -0,0 +1,322 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Methods for implementing the `datalab beta creategpu` command."""
+
+import os
+from builtins import input
+import subprocess
+import tempfile
+
+import create
+import connect
+import utils
+
+
+description = ("""`{0} {1}` creates a new Datalab instance running in a Google
+Compute Engine VM with a GPU.
+
+This command also creates the 'datalab-network' network if necessary.
+
+By default, the command creates a persistent connection to the newly
+created instance. You can disable that behavior by passing in the
+'--no-connect' flag.""")
+
+_NVIDIA_PACKAGE = 'cuda-repo-ubuntu1604_8.0.61-1_amd64.deb'
+_DATALAB_NETWORK = 'datalab-network'
+_DATALAB_NETWORK_DESCRIPTION = 'Network for Google Cloud Datalab instances'
+
+_DATALAB_FIREWALL_RULE = 'datalab-network-allow-ssh'
+_DATALAB_FIREWALL_RULE_DESCRIPTION = 'Allow SSH access to Datalab instances'
+
+_DATALAB_DEFAULT_DISK_SIZE_GB = 200
+_DATALAB_DISK_DESCRIPTION = (
+    'Persistent disk for a Google Cloud Datalab instance')
+
+_DATALAB_NOTEBOOKS_REPOSITORY = 'datalab-notebooks'
+
+_DATALAB_STARTUP_SCRIPT = """#!/bin/bash
```

Comment:
There's a lot of code duplicated from the `create.py` file here.

Can we move that off to something like a `base_startup_script` constant, and then have the two packages just define suffixes for the base startup script?